workshops/personal_website/README.md

```diff
@@ -163,331 +122,204 @@ Every HTML file follows this template:
     </body>
 </html>
 ```
+This structure is common to all HTML pages. If you want to find out more, just
+open up any website and inspect it. You'll find each of these elements on every
+page-- the doctype, and an HTML tag wrapped around a head and body.
+
+Before proceeding, we'll briefly go over what this template means. HTML works
+by storing information inside tags. `<html></html>` is an example of one such
+tag. Inside `<html></html>`, we've placed two other sets of tags:
+`<head></head>` (which wraps around the "head") and `<body></body>` (which
+wraps around the "body"). The body holds everything you would see in the actual
+tab/window when you open the page, while the head conveys information about the
+page to the browser.
+`<!DOCTYPE html>` tells the browser what version of HTML to expect. Since it is
+a language, HTML is constantly growing and updating, so there are multiple
+versions. In our case, we are going to use HTML5, the latest version.
+
+### 2) Previewing the Page
+
+Let's check out what our HTML file looks like in Live Preview! First, we'll
+save the file by clicking "File" → "Save" (or use the shortcut CTRL+s /
+Command+s). Then, preview what the website looks like by clicking "Preview" →
+"Live Preview File"
+
+As you can see, the page is blank. This is because we haven't added anything to
+the `body` section yet. Let's add some content!
+
+### 3) Adding Text to the Body
+
+As mentioned before, all information is wrapped in tags. Tags are predefined in
+the language; think of them as the words in the language. For text, HTML
+provides a number of tags to store text, two of the most basic ones being the
+heading1 tag (`<h1>`) and the paragraph tag (`<p>`). Just as with the other
```

Comment:
> heading1 tag

Probably should be "heading tag", and then an explanation that `h1` is the first level of heading.